package.json

```diff
@@ -53,7 +53,7 @@
     "tslint": "^4.0.0",
     "tslint-config-ktsn": "^2.0.0",
     "typescript": "^2.1.4",
-    "vue": "^2.1.10",
+    "vue": "^2.2.6",
```

Comment:
If I understand correctly, we must use `import Vue from 'vue'` syntax when using with webpack2 and Vue 2.4.
And Vue 2.2.0 does not accept this, Vue 2.2.6 does.

`^2.3.0` is better as you said.